mythril/laser/ethereum/svm.py

```diff
@@ -305,17 +308,17 @@ def execute_state(
             return [new_global_state], op_code
 
         except TransactionEndSignal as end_signal:
-            transaction, return_global_state = (
-                end_signal.global_state.transaction_stack.pop()
-            )
+            transaction, return_global_state = end_signal.global_state.transaction_stack[
+                -1
+            ]
```

Comment:
I'm slightly confused for the reasoning for this change. Why was popping from the transaction stack bad?

Reply:
Because the callback is issued after this pop.
That means that for a creation transaction it will receive a global state with a transaction stack of [], and it will be impossible to determine the transaction that the global state was from

Reply:
Ah. So is it then popped after the callback?

Reply:
I don't think it has to be popped at all (and I think it isn't)